src/schedule/ucc_schedule.h

```diff
@@ -41,26 +56,31 @@ typedef struct ucc_event_manager {
 } ucc_event_manager_t;
 
 enum {
-    UCC_COLL_TASK_FLAG_INTERNAL = UCC_BIT(0),
-    UCC_COLL_TASK_FLAG_CB       = UCC_BIT(1)
+    UCC_COLL_TASK_FLAG_INTERNAL      = UCC_BIT(0),
+    UCC_COLL_TASK_FLAG_CB            = UCC_BIT(1),
+    UCC_COLL_TASK_FLAG_EXECUTOR      = UCC_BIT(2),
+    UCC_COLL_TASK_FLAG_TOP_LEVEL     = UCC_BIT(3),
```

Comment:
plz add a brief description of new flags

Reply:
added